src/dmd/builtin.d

```diff
@@ -124,6 +124,13 @@ extern (C++) Expression eval_log10(Loc loc, FuncDeclaration fd, Expressions* arg
     return new RealExp(loc, CTFloat.log10(arg0.toReal()), arg0.type);
 }
 
+extern (C++) Expression eval_exp(Loc loc, FuncDeclaration fd, Expressions* arguments)
```

Comment:
Not my expertise but Walter has changed a lot of passing `Loc` by value to pass by `const ref`

Reply:
Ah, ok.